server.go

```diff
@@ -142,7 +146,47 @@ func handleProcessWebhook(c echo.Context) (err error) {
 	case webhook.PullRequestPayload:
 		switch payload.Action {
 		case "opened", "reopened", "synchronize":
-			res, err := ourGithub.HandlePullRequest(c.Logger(), postgresDB, payload, appId, getCurrentCLAVersion())
+			// Getting a JWT Apps Transport to ask GitHub about stuff that needs a JWT for asking, such as installInfo
+			atr, err := ghinstallation.NewAppsTransportKeyFromFile(http.DefaultTransport, int64(appId), filenameTheClaPem)
```

Comment:
I moved the transport setup up a level from `github`, because technically we can make these transports longer lived as well.